boards/arm/thingy91_nrf9160/board_nonsecure.c

```diff
@@ -7,8 +7,16 @@
 #include <init.h>
 
 #if defined(CONFIG_NRF_MODEM_LIB) && defined(CONFIG_NET_SOCKETS_OFFLOAD)
+#if defined(CONFIG_POSIX_API)
+#include <posix/unistd.h>
+#include <posix/netdb.h>
```

Comment:
Not a big deal, but do we need all these headers? We only have some basic socket operation in the file so it should be enough to get `posix/sys/socket.h` and `posix/unistd.h` for `close()`?

Reply:
Yep, you are correct. Those two are enough. I've removed other includes.